Classes/Private/Manager/Submanagers/OCTCallsContainer.m

```diff
@@ -0,0 +1,118 @@
+//
+//  OCTCallsContainer.m
+//  objcTox
+//
+//  Created by Chuong Vu on 6/10/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import "OCTCallsContainer.h"
+#import "OCTCallsContainer+Private.h"
+#import "OCTBasicContainer.h"
+
+@interface OCTCallsContainer () <OCTBasicContainerDelegate>
+
+@property (strong, nonatomic) OCTBasicContainer *container;
+
+@end
+
+@implementation OCTCallsContainer
+
+#pragma mark - Lifecycle
+
+- (instancetype)init
+{
+    self = [super init];
+
+    if (! self) {
+        return nil;
+    }
+
+    self.container = [[OCTBasicContainer alloc] initWithObjects:nil];
+    self.container.delegate = self;
+
+    return self;
+}
+#pragma Public Methods
+
+/**
+ * @return Number of calls
+ */
```

Comment:
I don't think it is good to duplicate documentation in implementation. Later you'll update it in one place, but will forget to update in another, etc.